.github/workflows/python-package.yml

```diff
@@ -0,0 +1,169 @@
+        # test dependencies
+        pip install --upgrade pytest pytest-cov pytest-benchmark IPython
+        
+    # conda
+    - name: Set up Python ${{ matrix.python-version }} (conda)
+      if: matrix.conda
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-update-conda: true
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies (conda)
+      if: matrix.conda
+      run: |
+        echo $CONDA/bin >> $GITHUB_PATH;
+        conda install \
+          "numpy>=1.17.0" \
+          scipy \
+          pip \
+          IPython \
+          h5py;
+        conda install -c conda-forge sparse;
+        conda install -c numba "numba>=0.45.1";
+        # make sure in conda we do not overwrite dependencies with pip
+        python setup.py develop --no-deps;
+        # test dependencies
+        python -m pip install --upgrade pytest pytest-cov pytest-benchmark;
+
+    - name: Test with pytest
+      env:
+        MODE: ${{ matrix.mode }}
+        NUMBA_DISABLE_JIT: ${{ matrix.disable_jit }}
+      run: |
+        PYTEST_ARGS=();
+        if [[ "${MODE}" == "bench" ]]; then
+          PYTEST_ARGS+=(--benchmark-only);
+        else
+          PYTEST_ARGS+=(--benchmark-skip);
+        fi;
+        if [[ "${MODE}" == "very_slow" ]]; then
+          PYTEST_ARGS+=(-m "veryslow");
+        else
+          PYTEST_ARGS+=(-m "not veryslow");
+        fi;
+        if [[ "${MODE}" == "doctests" ]]; then
+          PYTEST_ARGS+=(--doctest-modules --ignore clifford/test);
+        fi;
+
+        # `python -m` ensures we dispatch to conda if appropriate
+        python -m pytest \
+          "${PYTEST_ARGS[@]}" \
+          --color=yes \
+          -o junit_family=legacy \
+          --junitxml=junit/test-results.xml \
+          --durations=25 \
+          --cov=clifford \
+          --cov-branch;
+    # This only publishes result for the last run right now
+    # - name: Publish Test Report
+    #   uses: mikepenz/action-junit-report@v2
+    #   if: ${{ always() }}
+    #   with:
+    #     report_paths: 'junit/test-results.xml'
+    - uses: codecov/codecov-action@v1
+
+  deploy:
+    needs: test
+    runs-on: ubuntu-latest
+    name: deploy
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: "Install"
+      run: |
+        python -m pip install --upgrade pip;
+        python -m pip install build
+        python -m build --sdist --wheel --outdir dist/
+    - uses: actions/upload-artifact@v2
+      with:
+        name: dist
+        path: dist
+#     - uses: pypa/gh-action-pypi-publish@v1.4.2
+#       user: arsenovic
+#       password: ieUd3c2DjrZQE+3FlqmU5FQObNWIDiL9E9G6aLs0ksEKAi5Z1t7fefXic1XHsHOZZYteycef/lZkUYg3ijwfZg2xzELeTdLef29GgUrxYuGL4MJ706UFj450Xlv9l1oH5D32OEKT2EwfxMqdrw39+N8zD5ehVyQbYM6Z3lwtCvg=
```

Comment:
I'm pretty sure this won't work, as the password is encrypted with a key that only travis knows, hence it being commented out.